addons/account_peppol/tests/test_peppol_messages.py

```diff
@@ -266,6 +266,19 @@ def test_send_success_message(self):
         )
         self.assertTrue(bool(move.ubl_cii_xml_id))
 
+    def test_send_peppol_and_email_default_values(self):
+        # If both "Send by Email" and "Send by Peppol" are set, we deactivate the "Send by Email" option
+        move = self.create_move(self.valid_partner)
+        move.action_post()
+
+        wizard = self.create_send_and_print(
+            move,
+            checkbox_send_peppol=True,
+        )
+
+        self.assertTrue(wizard.checkbox_send_peppol)
+        self.assertFalse(wizard.checkbox_send_mail)
```

Comment:
Your test is working by chance: `checkbox_send_mail` is False regardless of Peppol's state since your have no email set on the partner.